downstream/modules/automation-hub/proc-obtaining-org-collection-url.adoc

```diff
@@ -3,23 +3,25 @@
 [id="proc-create-api-token"]
 = Retrieving your Red Hat Certified Collections Sync URL and API token.
 
-You can sync Red Hat certified collections curated by your organization from cloud.redhat.com to your local Automation Hub.
+You can sync Red Hat certified collections curated by your organization from console.redhat.com to your local {HubName}.
 
 .Prerequisites
 
-* You have Organization Administrator permissions for cloud.redhat.com.
+* You have Organization Administrator permissions for console.redhat.com.
 
 .Procedure
 
-. Log in to cloud.redhat.com as an Organization Administrator.
-. Navigate to *Automation Hub* -> *Repo Management*.
-. Locate the *Sync URL* and click image:images/copy.png[Copy to clipboard,10,25]. Paste the *Sync URL* in a file to use when configuring the rh-certified remote.
-. Click image:images/more_actions.png[More,10,25] and click *Get token*.
-. On the *Token management* page, click *Load token*.
-. Click *Copy to clipboard* to copy the API token.
+. Log in to console.redhat.com as an Organization Administrator.
+. Navigate to menu:Automation Hub[Repo Management].
+. Locate the *Sync URL* and click image:images/copy.png[Copy to clipboard,10,25]. 
+Paste the *Sync URL* in a file to use when configuring the rh-certified remote.
+. Click image:images/more_actions.png[More,10,25] and click btn:[Get token].
```

Comment:
The more actions icon doesn't look very clear in my docs build. Would it make sense to say something like
```suggestion
. Click the btn:[more actions] icon (image:images/more_actions.png[More,10,25]) and click btn:[Get token].
```

Reply:
The icons are pretty awful in a bccutil build too, but look OK in the published doc.